includes/core-changes/aspnetcore/5.0/iis-urlrewrite-middleware-query-strings-are-preserved.md

```diff
@@ -0,0 +1,50 @@
+### IIS: UrlRewrite middleware query strings are preserved
+
+An IIS UrlRewrite middleware defect prevented the query string from being preserved in rewrite rules. That defect has been fixed to maintain consistency with the IIS UrlRewrite Module's behavior.
+
+For discussion, see issue [dotnet/aspnetcore#22972](https://github.com/dotnet/aspnetcore/issues/22972).
+
+#### Version introduced
+
+ASP.NET Core 5.0 Preview 7
+
+#### Old behavior
+
+Consider the following rewrite rule:
```

Comment:
Per the [style guide](https://styleguides.azurewebsites.net/Styleguide/Read?id=2700&topicid=33791), 

>Don't use foo, foobar, or related words in content for a general audience. It's OK to use these words as placeholders or variable names in content for a technical audience, but if you can, use another placeholder instead, such as Example.exe.